client/src/ActiveSessionPoemListContainer.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { PropTypes } from 'react';
+import Sortable from 'sortablejs';
+import { requestWithAuth } from './auth';
+import SessionSortablePoemList from './SessionSortablePoemList';
+
+class ActiveSessionPoemListContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      session: {
+        poems: [],
+      },
+      intervalId: -1,
+    };
+  }
+
+  componentDidMount = () => {
+    this.loadActiveSession();
+    this.state.intervalId = setInterval(this.loadActiveSession, this.props.route.pollInterval);
+    const el = document.getElementById('activeSessionPoemList');
+    Sortable.create(el, {
+      onUpdate: (evt) => {
+        const reordering = this.state.session.poems.slice();
+        const movedPoem = reordering.splice(evt.oldIndex, 1);
+        reordering.splice(evt.newIndex, 0, ...movedPoem);
```

Comment:
We probably want to ensure that exactly one poem is moved. But `Array.splice` returns an array, so it's either `...movedPoem` or `movedPoem[0]`. This seems better given any lack of checking (ie, what if `moedPoems` is empty?)